packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1352,3 +1355,171 @@ export const listProjectsTask = async ({}, hre: HardhatRuntimeEnvironment) => {
 task(TASK_CHUGSPLASH_LIST_PROJECTS)
   .setDescription('Lists all projects that are owned by the caller.')
   .setAction(listProjectsTask)
+
+const claimOwnershipTask = async (
+  args: {
+    configPath: string
+    referenceName: string
+    silent: boolean
+  },
+  hre
```

Comment:
We can set this type to be `HardhatRuntimeEnvironment`. Same with the transferOwnershipTask

Reply:
Looks like it wasn't added to the `transferOwnershipTask`